NEWS.md

```diff
@@ -1,3 +1,39 @@
+# epichains 0.1.0
+
+## Package name change
+
+* `epichains` is a re-implementation of `bpmodels` with a focus on providing
+a dedicated class of data structures for easy manipulation and interoperability
+with other new tools in the pipeline.
+
+### Features
+
+* `simulate_tree()`: simulate transmission trees from a given number of chains.
+* `simulate_tree_from_pop()`: simulate transmission trees from a given number 
+  population size and initial immunity.
+* `simulate_vect()`: simulate a vector of observed transmission chains 
```

Comment:
will need to be updated

Reply:
I've fixed that and cleaned up the changelog here 580a98b02552e9c00294d92bcfe6e38f633a9c8b.